examples/ros2-basic.nix

```diff
@@ -8,9 +8,11 @@ mkShell {
   nativeBuildInputs = [
     (buildEnv {
       paths = [
-        ros-core
         colcon
-        geometry-msgs
+        ros-core
+        rviz2
+        ros2bag
+        sensor-msgs
```

Comment:
I think we can update the example, but I'd do it in a separate commit with an explanation why.

And I'm not sure whether we want to include `rviz2` in the basic example. rviz seems to be broken quite often so I'd prefer to have a simple example with CLI tools only, which demonstrates basic functionality and works most of the time. Then, we can add a more elaborate example, say `ros2-desktop`, which would contain the [dekstop](https://www.ros.org/reps/rep-2001.html#id25) entry point.

Reply:
Sorry it's a mistake, forgot to remove that after testing, will revert that